catalog/lib/widgets/app_select_dropdown.dart

```diff
@@ -0,0 +1,165 @@
+import 'package:catalog/catalog.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+typedef MenuItems<T> = ({T value, String label});
```

Comment:
Shouldn't be plural? Shouldn't it be a class? Because we may need equal and hashcode

Reply:
The records equality is field by field. https://api.flutter.dev/flutter/dart-core/Record/operator_equals.html.
So if T has a well defined equal then it should work fine